lib/screens/e_commerce/myOrder.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:promdi_fe/screens/e_commerce/order_list.dart';
+
+class MyOrder extends StatefulWidget {
+  const MyOrder({Key? key}) : super(key: key);
+
+  @override
+  _MyOrderState createState() => _MyOrderState();
+}
+
+class _MyOrderState extends State<MyOrder> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: DefaultTabController(
+        length: 3,
+        child: Scaffold(
+          appBar: AppBar(
+            automaticallyImplyLeading: false,
+            flexibleSpace: Column(
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: [
+                TabBar(
+                  tabs: [
+                    Tab(
+                      text: 'Completed',
+                    ),
+                    Tab(
+                      text: 'On Progress',
+                    ),
+                    Tab(
+                      text: 'Canceled',
+                    ),
+                  ],
+                )
+              ],
+            ),
+          ),
+          body: Container(
+            child: TabBarView(
+              children: [
+                OrderList(),
+                OrderList(),
+                OrderList(),
```

Comment:
In the plan, we should be considering calling OrderList with a value to identify the category in which the order list should fall (i.e. 'Completed', 'In Progress', 'Canceled').